frontend-ssr/components/notifications-tooltip/loading-notification/LoadingNotification.component.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Let's rename to `NotificationSkeleton.tsx` - to keep the same naming convention as in `ExerciseSubmissionSkeleton`.

Reply:
I see you renamed the file, kudos for that. But I kinda expected you to also re'arrange the folders in a similar fashion with ExerciseSubmission. What I mean is:

```
📁 notification-tooltip
      NotificationTooltip.tsx
      📁notification
           Notification.tsx
           NotificationSkeleton.tsx
           Notification.module.scss
```

I know this may not seem like a big deal, and if it were just us coding it probably wouldn't be. But since I expect more contributions, it's important to have the same structure everywhere.